src/Serializer/SerializerHandlerInterface.php

```diff
@@ -13,17 +13,30 @@
 
 namespace Sonata\Serializer;
 
-use JMS\Serializer\Handler\SubscribingHandlerInterface;
+if (!interface_exists(\Sonata\Form\Serializer\SerializerHandlerInterface::class, false)) {
+    @trigger_error(
+        'The '.__NAMESPACE__.'\SerializerHandlerInterface class is deprecated since version 3.19.0 and will be removed in 4.0.'
```

Comment:
```suggestion
        'The '.__NAMESPACE__.'\SerializerHandlerInterface class is deprecated since sonata-project/core-bundle 3.19.0 and will be removed in version 4.0.'
```